service/internal/checksum/algorithms.go

```diff
@@ -30,6 +30,9 @@ const (
 
 	// AlgorithmSHA256 represents SHA256 hash algorithm
 	AlgorithmSHA256 Algorithm = "SHA256"
+
+	// AlgorithmCRC64NVME represents CRC64NVME hash algorithm
+	AlgorithmCRC64NVME Algorithm = "CRC64NVME"
```

Comment:
Interesting, so this is used only when someone computes their own CRC64NVME checksum and pass their own implementation?

We should add a test with some pre-computed values to make sure we don't panic trying to use an algorithm that we don't have an implementation for

Reply:
s3 team already manual test that, but I can add that

Reply:
yeah, I don't think S3 will panic, I want to ensure we don't have any logic on the SDK to do something like "if CRC64 try to compute checksum" and then we panic